### DISCUSSION TITLE
OAS - Json keys ordering

### DISCUSSION THREAD

**Discussion by samuelecarpene** [directus/directus#9050] [Feature Requests]
I'm starting a discussion from an issue that i opened.

https://github.com/directus/directus/issues/9046

The main goal is to have json or yaml generated from /server/specs/oas that can be conpared in time.

If specifications say that json is an unorderded collection of zero or more name/value pairs but does anyone forbid to order them?

**Comment by rijkvanzanten** [comment]
We rely on `swagger-cli` to bundle the static specs into JSON, which is then adjusted by the API to form the final spec. As far as I'm aware, if you generate the spec twice, the output is the same.

What keys are consistently reordered that are causing troubles?

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
